### ISSUE TITLE
E: Footer is wrong

### ISSUE THREAD

**Issue by luguenth** [mrded/nokia-8110-root#1]
Maybe its outdated already. But I'm getting the ErrorMessage 
`E: footer is wrong`
`E: signature verification failed`

**Comment by mrded** [comment]
Try to roll back your changes to `f012ec609dafb5d1fe7e24788a3055baa8d511c7` commit. Just curious if it's working.

**Comment by luguenth** [comment]
Already rooted the Phone with the [telnet-Method](https://sites.google.com/view/bananahackers/root/temporary-root)

**Comment by Loris1123** [comment]
I'm having the same issue. The version at `f012ec609dafb5d1fe7e24788a3055baa8d511c7` is also not working for me

**Comment by elbfbt** [comment]
yeah i have the same problem on my sm-t700 when updating from sd card ( im updating from sd card bc my pc is not detecting my phone in odin mode at all....help
